src/main/docker/kafdrop.sh

```diff
@@ -64,5 +64,4 @@ ARGS="--add-opens=java.base/sun.nio.ch=ALL-UNNAMED -Xss256K \
      $HEAP_ARGS \
      $JVM_OPTS"
 
-java $ARGS -jar /kafdrop*/kafdrop*jar $CMD_ARGS
-
+exec java $ARGS -jar /kafdrop*/kafdrop*jar ${CMD_ARGS}
```

Comment:
Do we really need `exec` here to replace the pid?

Reply:
Docker container is designed to run single process. Run of many processes can be supported by init system like [tini](https://github.com/krallin/tini) that handles signals and sends them to child process. Bash scripts for example do not handle and emit signals properly.

I think we need to replace process PID to 1 here so that the java application can handle the SIGTERM/SIGKILL signals sent to the container.

Usfull link: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint
> This script uses the exec Bash command so that the final running application becomes the container’s PID 1. This allows the application to receive any Unix signals sent to the container. > For more, see the ENTRYPOINT reference.